pkg/network/ebpf/c/protocols/http2/maps-defs.h

```diff
@@ -11,7 +11,7 @@ BPF_HASH_MAP(http2_static_table, u8, static_table_value_t, 20)
 
 /* http2_dynamic_table is the map that holding the supported dynamic values - the index is the static index and the
    conn tuple and it is value is the buffer which contains the dynamic string. */
-BPF_LRU_MAP(http2_dynamic_table, dynamic_table_index_t, dynamic_table_entry_t, 0)
+BPF_LRU_MAP(http2_dynamic_table, dynamic_table_index_t, dynamic_table_entry_t, !BPF_F_NO_COMMON_LRU)
```

Comment:
Are you sure this is doing what you think ? AFAIK `BPF_F_NO_COMMON_LRU == (1U << 1) == 2`,
So `!BPF_F_NO_COMMON_LRU == 0` no ? So basically no change

Reply:
Moreover, I believe the last argument of `BPF_LRU_MAP` is actually proviing the max entries, not the flag of the given map

Reply:
Finally I believe the shared LRU between CPU cores is actually the default, so I would expect you already have the desired behavior